### ISSUE TITLE
[PHP] Any way to use WKT Struct?

### ISSUE THREAD

**Issue by vinceferro** [protocolbuffers/protobuf#4438]
I'm having troubles at creating an object given this proto definition:

```
message Session {
  SessionId session_id = 1;
  google.protobuf.Struct session_data = 2;
}
```

Is there any way to convert a php object/array to a Struct?
In go I use jsonpb, but I couldn't find a way to do the same thing in PHP.

Thanks

**Comment by TeBoring** [comment]
Currently no. We are working on that.

**Comment by vinceferro** [comment]
Ah ok I see, thanks!

**Comment by TeBoring** [comment]
Should have been fixed in #5106. You can use Struct to parse a json array.
